Readme.md

```diff
@@ -0,0 +1,51 @@
+# BeCJK & History
+
+![screenshot](screenshot.png)
+
+BeCJK, is an input method for the legancy BeOS, first Developed by
+Anthony.Lee.
+
+Now, this copy of code is gotten from Andraw Bachmann.
+
```

Comment:
This copy was obtained from Andrew Bachmann.

Reply:
I will change it.